packages/common/src/account/kms/kmsKeyToAccount.ts

```diff
@@ -19,11 +20,15 @@ export type KmsAccount = LocalAccount<"aws-kms"> & {
  * @returns A Local Account.
  */
 export async function kmsKeyToAccount({
-  keyId,
+  keyId = process.env.AWS_KMS_KEY_ID,
   client = new KMSClient(),
-}: KmsKeyToAccountOptions): Promise<KmsAccount> {
+}: KmsKeyToAccountOptions = {}): Promise<KmsAccount> {
   const address = await getAddressFromKms({ keyId, client });
 
+  if (!keyId) {
+    throw new MUDError("Missing AWS KMS Key ID");
+  }
```

Comment:
I think this check and error should be done inside CLI not here, and leave `keyId` as a required prop